viewer/packages/camera-manager/src/ComboControls.ts

```diff
@@ -41,14 +41,13 @@ function getPinchInfo(domElement: HTMLElement, touches: TouchList) {
 const defaultPointerRotationSpeed = Math.PI / 360; // half degree per pixel
 const defaultKeyboardRotationSpeed = defaultPointerRotationSpeed * 10;
 
-export default class ComboControls extends EventDispatcher {
+export class ComboControls extends EventDispatcher {
   public enabled: boolean = true;
   public enableDamping: boolean = true;
   public dampingFactor: number = 0.25;
   public dynamicTarget: boolean = true;
   public minDistance: number = 0.1;
-  public maxDistance: number = Infinity;
-  public dollyFactor: number = 0.98;
+  public dollyFactor: number = 0.99;
```

Comment:
Why is maxDistance removed and dollyFactor introduced?

Reply:
Reason for removal of maxDistance is described above, dollyFactor default is taken from the value that was before set every time ComboControls were created in Cognite3DViewer. Also, this difference in dolly factor (0.01) affects controls in very tiny way, so doesn't really matter here. It is now just defaulted to the value that it was always used with before.